packages/core/src/application.ts

```diff
@@ -88,6 +91,9 @@ export class Application extends Context {
     response: ServerResponse,
   ) {
     this._setupHandlerIfNeeded();
+    if (request.method === 'GET' && request.url === '/openapi.json') {
+      return this._serveOpenApiSpec(request, response);
```

Comment:
This is a bit hacky/ugly. Here are the reasons why I decided to not register a new `Route` instance instead:

1. Regular routes are handled through `Sequence`. IMO, this built-in endpoint should not run through a `Sequence`, because it's not part of the application API itself. E.g. if the app implements access/audit logs, I don't want this endpoint to trigger a log entry. If the server implements content-negotiation to support XML clients, I don't want the OpenAPI spec to be converted into XML response.

2. Regular routes are included in OpenAPI spec. As a result, `GET /openapi.json` would be described in the OpenAPI spec too. I suppose this can be addressed by introducing some sort of a `hidden` flag to `RouteEntry` object, or maybe it's ok to include this endpoint in API spec?

3. In `_serverOpenApiSpec`, I want a better control of how the response is created. For example, we may want to use a different content-type than `application/json` (is there any OpenAPI-Spec type registered yet?). At the moment, LB-Next does not allow route handlers/controller methods to modify response headers.